dev/breeze/src/airflow_breeze/params/doc_build_params.py

```diff
@@ -20,20 +20,30 @@
 from dataclasses import dataclass
 
 from airflow_breeze.branch_defaults import AIRFLOW_BRANCH
+from airflow_breeze.global_constants import ALL_SPECIAL_DOC_KEYS
 
 providers_prefix = "apache-airflow-providers-"
 
 
-def get_provider_name_from_short_hand(short_form_providers: tuple[str]):
+def get_provider_name_from_short_hand(short_form_packages: tuple[str, ...]) -> tuple[str, ...]:
```

Comment:
Should we instead make the type fixed instead of using elipses?

Reply:
No. This is a problem mypy recently pointed at in the code `tuple[str]` behaves differently than `list[str]` - this is a special case. Unlike list[str]. The `tuple[str]` means tuple with single str element, `tuple[str, ...]` means tuple with any number of str elements

See https://docs.python.org/3/library/typing.html#annotating-tuples